src/htmlrenderer.cpp

```diff
@@ -761,12 +777,9 @@ void HtmlRenderer::render(std::istream& input,
 				break;
 
 			case HtmlTag::SCRIPT:
-				// don't render scripts, ignore current line
-				if (inside_script) {
-					inside_script--;
-				}
-				prepare_new_line(curline,
-					tables.size() ? 0 : indent_level);
+				// This line is unreachable, since we handle closing <script>
+				// tags way before this entire `switch`.
+				assert(false);
```

Comment:
With correct html input, this is indeed unreachable.

However, if an rss feed contains invalid html (e.g. a closing `</SCRIPT>` tag without an opening `<SCRIPT>` tag), this can still be reached.

Reply:
You're right. I'll drop the `assert` then. Thanks!